### PR TITLE
Feature: Add option for taking transparent screenshots

### PR DESCRIPTION
**Pull request by CorySanin**

### CODE REVIEW

File: src/openrct2/interface/Screenshot.h [hunk 1]
```diff
@@ -49,6 +49,7 @@ struct CaptureOptions
     std::optional<CaptureView> View;
     ZoomLevel Zoom;
     uint8_t Rotation{};
+    bool Transparent;
```
Comment (IntelOrca) [inline src/openrct2/interface/Screenshot.h:52]:
```suggestion
    bool Transparent{};
```

File: distribution/openrct2.d.ts [hunk 2]
```diff
@@ -314,6 +314,11 @@ declare global {
          * Rotation of the camera from 0 to 3.
          */
         rotation: number;
+
+        /**
+         * Whether to enable transparency in the screenshot
```
Comment (IntelOrca) [inline distribution/openrct2.d.ts:319]:
```suggestion
         * Whether to enable transparency in the screenshot.
```

**Review by IntelOrca — CHANGES_REQUESTED** [review]
Plugin API version should be incremented.

**Review by IntelOrca — APPROVED** [review]

**Comment by CorySanin** [comment]
Can someone re-run the Docker check? Looks like some kind of fluke. My Docker image builds fine locally.

**Pull request merged**